### ISSUE TITLE
Message: Unable to add lists to 'message-text' for screen readers

### ISSUE THREAD

**Issue by chaseerickson** [infor-design/enterprise#4400]
**Describe the bug**
When creating a Message component, adding `<ol>` or `<ul>` to the 'allowedTags' option allows for these elements to appear in the message, however they get placed outside the ‘message-text’ `<p>`, which results in them not being read by screen readers. Other elements, such as `<span>`, appear inside the ‘message-text’ `<p>` when they are added to the ‘allowedTags’ option, and are successfully read by screen readers. Both of these scenarios adhere to HTML spec, however this structure prevents lists inside messages from adhering to accessibility guidelines.

**To Reproduce**
1. Create a Message component with parameters similar to the following: `allowedTags: '<li><ol>', message: '<ol><li>Test</li></ol>'`
2. With a screen reader on, observe that the "Test" list appears in the message, but is not read by the screen reader.

**Expected behavior**
Content in the 'message-text' area of a Message component should be successfully read by screen readers.

**Version**

- ids-enterprise: v4.28.0

**Screenshots**
Below in **Additional context**

**Platform**
- Infor Application/Team Name: LSF
- Device: Laptop
- OS Version: Windows 10
- Browser Name: Chrome
- Browser Version: 85.0.4183.83 (Official Build) (32-bit)

**Additional context**
Perhaps the ‘message-text’ `<p>` could be changed to a ‘message-text’ `<div>`, or the ‘message-text’ ID could be changed to a class so more than one element can use it? Here are three examples, with the third one being the issue:

Scenario 1 (without allowedTags, everything gets converted to `<p>`):

CODE:
 
![image](https://user-images.githubusercontent.com/13371173/93804778-64d96f80-fc0c-11ea-9fce-dd8abae73b0c.png)

RESULT:
 
![image](https://user-images.githubusercontent.com/13371173/93804797-6a36ba00-fc0c-11ea-89d9-44a14fbad45c.png)

Scenario 2 (with allowedTags, `<span>` gets placed inside ‘message-text’ `<p>`):

CODE:
 
![image](https://user-images.githubusercontent.com/13371173/93804815-7458b880-fc0c-11ea-8a89-acccf9ce7b86.png)

RESULT:
 
![image](https://user-images.githubusercontent.com/13371173/93804834-79b60300-fc0c-11ea-8087-17c84aed4abd.png)

Scenario 3 (with allowedTags, `<ol>` gets placed outside ‘message-text’ `<p>`):

CODE:
 
![image](https://user-images.githubusercontent.com/13371173/93804885-85092e80-fc0c-11ea-9a73-4e7951329563.png)

RESULT:

![image](https://user-images.githubusercontent.com/13371173/93805716-b33b3e00-fc0d-11ea-8fcc-b1bf258cb2b3.png)

**Comment by tmcconechy** [comment]
Does just moving the ol into the p make the screen reader work ok? I feel like if its in the DOM that shouldnt matter if its before or after? But for styling it should be in the `p`

**Comment by chaseerickson** [comment]
Yes I tried manually appending the `<ol>` inside the `<p>` and it was successfully read by the screen reader (example below), however according to HTML spec, `<ol>` elements are not legally allowed inside `<p>` elements.

CODE:

![image](https://user-images.githubusercontent.com/13371173/93808616-d8ca4680-fc11-11ea-8228-d9358d952ec6.png)


RESULT

![image](https://user-images.githubusercontent.com/13371173/93808648-e41d7200-fc11-11ea-9545-e1be62440f0a.png)

**Comment by tmcconechy** [comment]
Maybe the <p> should be a <div> as to be more "Generic"

**Comment by chaseerickson** [comment]
Yes when I remove the `<p>` and place the 'message-text' id on the '.model-body' `<div>`, the child `<ol>` is successfully read by the screen reader, so changing the `<p>` to a `<div>` should be okay:

CODE:

![image](https://user-images.githubusercontent.com/13371173/93924444-5c4f6a80-fcda-11ea-92be-6277c9f29b2f.png)

RESULT:

![image](https://user-images.githubusercontent.com/13371173/93924463-64a7a580-fcda-11ea-90c2-6cf00cd0fea6.png)

**Comment by jbrcna** [comment]
**QA Passed**
v4.34.0-dev
https://master-enterprise.demo.design.infor.com/components/message/test-lists.html
![image](https://user-images.githubusercontent.com/56722184/95197182-cce2a500-080b-11eb-8e90-1e309a958f59.png)
